src/main/java/com/baidu/hugegraph/logger/AuditLogger.java

```diff
@@ -0,0 +1,548 @@
+    }
+
+    /**
+     * Finalize health check
+     * @param serviceId
+     */
+    public void logFinalizeHealthCheck(String serviceId) {
+        infoLogger
+        .generalLogMessage(LogTemplate.SERVICE_HEALTH_CHECK_RESULT, serviceId);
+    }
+
+    /**** Instance related logs ****/
+    /**
+     * Create new instance
+     * @param instanceId
+     */
+    public void logCreateInstance(String instanceId, String executorId) {
+        infoLogger
+        .generalLogMessage(LogTemplate.CREATE_INSTANCE, instanceId);
+    }
+
+    /**
+     * Delete instance
+     * @param instanceId
+     */
+    public void logDeleteInstance(String instanceId, String executorId) {
+        infoLogger
+        .generalLogMessage(
+            LogTemplate.DELETE_INSTANCE, instanceId, executorId);
+    }
+
+    /**
+     * Start instance
+     * @param instanceId
+     */
+    public void logStartInstance(String instanceId, String executorId) {
+        infoLogger
+        .generalLogMessage(
+            LogTemplate.START_INSTANCE, instanceId, executorId);
+    }
+
+    /**
+     * Stop instance
+     * @param instanceId
+     */
+    public void logStopInstance(String instanceId, String executorId) {
+        warnLogger
+        .generalLogMessage(
+            LogTemplate.STOP_INSTANCE, instanceId, executorId);
+    }
+
+    /**
+     * Restart instance
+     * @param instanceId
+     */
+    public void logRestartInstance(String instanceId, String executorId) {
+        warnLogger
+        .generalLogMessage(
+            LogTemplate.RESTART_INSTANCE, instanceId, executorId);
+    }
+
+    /**
+     * Synchronize instance config
+     * @param instanceId
+     * @param executorId
+     */
+    public void logSyncInstanceConfig(String instanceId, String executorId) {
+        infoLogger
+        .generalLogMessage(
+            LogTemplate.SYNC_INSTANCE_CONFIG, instanceId, executorId);
+    }
+
+    /**
+     * Instance begins to provide service
+     * @param instanceId
+     * @param serviceId
+     */
+    public void logProvideService(String instanceId, String serviceId) {
+        infoLogger
+        .generalLogMessage(
+            LogTemplate.INSTANCE_PROVIDE_SERVICE,
+            instanceId,
+            serviceId);
+    }
+
+    /**
+     * Instance withdraws service from cluster
+     * @param instanceId
+     * @param serviceId
+     */
+    public void logWithdrawService(String instanceId, String serviceId) {
+        warnLogger
+        .generalLogMessage(
+            LogTemplate.INSTANCE_WITHDRAW_SERVICE,
+            instanceId,
+            serviceId);
+    }
+
+    /**
+     * Export instance log
+     * @param instanceId
+     * @param executorId
+     */
+    public void logExportInstanceLog(String instanceId, String executorId) {
+        infoLogger
+        .generalLogMessage(LogTemplate.EXPORT_INSTANCE_LOG, executorId);
+    }
+
+    /**** Host related logs ****/
+
```

Comment:
will add corresponding log in the future？

Reply:
yes